client/packages/react/tsconfig.json

```diff
@@ -1,21 +1,10 @@
 {
+  "extends": "../../tsconfig.base.json",
```

Comment:
This extend is complaining that there is no `tsconfig.base.json`. Is it possible that base json is not part of final artifact when this extends in packages requires it to be? 

<img width="874" alt="Screenshot 2024-10-31 at 08 20 41" src="https://github.com/user-attachments/assets/f565f94e-d2c1-4ddc-9dc4-bca2f3a18aea">
<img width="336" alt="Screenshot 2024-10-31 at 08 21 01" src="https://github.com/user-attachments/assets/ff14fdf5-7a99-442d-bf56-e9298df76e75">